libs/live-dmk/src/index.tsx

```diff
@@ -63,6 +78,69 @@ export const useDeviceSessionState = (): DeviceSessionState | undefined => {
   return sessionState;
 };
 
+export const useDeviceSessionRefresherToggle = (ff: boolean) => {
```

Comment:
[SHOULD] rename first arg to `enabled`

Reply:
actually not sure i'm going to keep it as it's not used anywhere

Reply:
done